### ISSUE TITLE
Min width of main window too large

### ISSUE THREAD

**Issue by sebastianpech** [GitMurf/masonry-vanilla#8]
I guess this is actually a roam issue, but I think it would fit nicely into the features of masonry. The flex-box width of the main roam view seems to be fixed to a minimum of 40% (on my monitor at least, don't know how those 40% are calculated).

![image](https://user-images.githubusercontent.com/13043943/109395625-aabe0080-792d-11eb-845e-c23718df28f1.png)

This result in a lot of white space around the actual blocks in roam main. Maybe you can add resizing of the main window similarly to sidebar cards. Or somehow bypass the scaling restriction.

**Comment by sebastianpech** [comment]
At the moment I just set this manually in full-screen with
```
.roam-main {
	flex: 0 0 25% !important;	  
}
#right-sidebar {
	flex: 0 0 75% !important;	  
}
```

**Comment by orkhan10** [comment]
This is wasted space right here. I if there was a solution for this padding issue, The main window size could stay like original without wrapping in this configuration. 
The thing is I love the size of the main window, how small it is. When I play with the padding it messes up with the size of the main window.

![Screenshot 2021-02-28 at 21 55 27](https://user-images.githubusercontent.com/18570936/109433252-bb46a780-7a0f-11eb-9152-0c107f303e97.png)

**Comment by sebastianpech** [comment]
Get's quite bad on an ultra-widescreen:
![image](https://user-images.githubusercontent.com/13043943/109458475-25d20480-7a5d-11eb-9f5b-9b0021dedad2.png)

**Comment by orkhan10** [comment]
Are you using the js scripts of the masonry? 
![Screenshot 2021-03-01 at 12 33 35](https://user-images.githubusercontent.com/18570936/109491810-58debd00-7a8a-11eb-9216-50592bf67f44.png)

**Comment by sebastianpech** [comment]
Yes. This screenshot is from a test db where I don't use it. Issue is the same.
Can't make the left main window smaller than that.

**Comment by orkhan10** [comment]
Alright. Just wanted to make sure.
